katello/hooks/boot/01-helpers.rb

```diff
@@ -52,6 +57,10 @@ module HookContextExtension
   def param_value(mod, name)
```

Comment:
Can you move the `FIXME` from line 55 to just above this function? Otherwise someone might remove `local_postgresql?` as well. The `unless Kafo::HookContext.method_defined?(:param_value)` at the end of the file is also a risk.

Perhaps it's cleaner to add a separate module for things that shouldn't be upstreamed to Kafo itself.